recipes/reviewer/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2.7" %}
+{% set sha256 = "7b77536f6c3cda16e08f53eba44d3329bb9bc44356f04644a765b6fc79fdbd31" %}
+
+package:
+  name: reviewer
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/Illumina/REViewer/releases/download/v{{ version }}/REViewer-v{{ version }}-linux_x86_64.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
The downloaded file says `x86_64`. And it brings pre-built binary for x86_64. It is not really a `noarch` package!

Reply:
IMO it would be better to download the source tarball and build it instead of using the pre-built one.
This way you can try to make the package available for Mac OS (x86_64) and Linux ARM64.

Reply:
I agree that it would be the best solution, but since I've never complied C code myself before, I'm not really sure how to get started. I thought that downloading the pre-compiled binary would be the easiest way to get a working recipe (which could then be improved later).

Reply:
In that case you may go back to the pre-built binary and let some other user to improve it later.